src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -57,109 +53,53 @@ public ItemsView(IEnumerable itemsSource) : base()
         /// <param name="itemsSource">item's data source</param>
         /// <param name="layouter">item's layout manager</param>
         /// <param name="template">item's view template with data bindings</param>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+       [EditorBrowsable(EditorBrowsableState.Never)]
         public ItemsView(IEnumerable itemsSource, ItemsLayouter layouter, DataTemplate template) : base()
         {
             ItemsSource = itemsSource;
             ItemsTemplate = template;
             ItemsLayouter = layouter;
         }
 
-        /// <summary>
-        /// Scrolling Direction.
-        /// </summary>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
-        public new Direction ScrollingDirection
-        {
-            get
-            {
-                return base.ScrollingDirection;
-            }
-            set
-            {
-                base.ScrollingDirection = value;
-            }
-        }
-
-        private IEnumerable _itemsSource = null;
         /// <summary>
         /// Item's source data.
         /// </summary>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public IEnumerable ItemsSource
-        {
-            get
-            {
-                return _itemsSource;
-            }
-            set
-            {
-                _itemsSource = value;
-            }
-        }
-
-        /// <summary>
-        /// Internal encapsulated items data source.
-        /// </summary>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
-        internal IItemSource InternalItemSource;
+        public IEnumerable ItemsSource { get; set; }
```

Comment:
How about changing this to public virtual and let CollectionView override it?
Or changing this to GetItemsSource and SetItemsSource and let CollectionView override it?

Otherwise, the following issue may happen.
```
public void Func(ItemsView itemsView, ItemsSource itemsSource)
{
    itemsView.ItemsSource = itemsSource;
}
```
```
Func(collectionView, itemsSource);
//collectionView.ItemsSource != itemsSource
```

Reply:
I agree with that. you can also use `public abstract`.

Reply:
원래 virtual이었다가 internalsource 쪽 내용을 바꾸면서 문제가 있어서 다 new로 돌렸었는데 그 문제가 해결되고 나서도 그냥 그대로 뒀었네요.. virtual -> override가 맞을거 같습니다.